backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/TestCorrectionStatus.java

```diff
@@ -0,0 +1,5 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+public enum TestCorrectionStatus {
+    ORIGINAL, CORRECTED, REMOVED
```

Comment:
It might make more sense to make the output encoding a property of the enum, that way we wouldn't risk this getting out of sync with the export model definition.